x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/predicate/fulltext/FullTextUtils.java

```diff
@@ -26,7 +26,7 @@
             return emptyMap();
         }
         String[] list = Strings.delimitedListToStringArray(options, DELIMITER);
-        Map<String, String> op = new LinkedHashMap<String, String>(list.length);
+        Map<String, String> op = new LinkedHashMap<>(list.length);
```

Comment:
Maybe it makes sense to do validation and see whether the user repeats some options and throw an error in this case? It's a corner-case but one that can occur nevertheless.

Reply:
This check is done already here: https://github.com/elastic/elasticsearch/pull/33828/files#diff-d426dc4022ca6fa5fe5bcfe8b9373928R39

Reply:
Added some unit tests for that: https://github.com/elastic/elasticsearch/pull/33828/commits/32075e6787b68a573e546bdf8a58dbb2884fdb14

Reply:
:+1: